spec/controllers/reviews_controller_spec.rb

```diff
@@ -93,6 +92,11 @@ def valid_session
       assigns(:feedbacks).should eq([@feedback_sub])
     end
 
+    it "should call responder's xlsx method" do
+      get :summary, {id: @review.to_param, format: :xlsx}, valid_session
+      response.content_type.should include 'openxmlformats'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.